O/oxigraph_server/build_tarballs.jl

```diff
@@ -3,27 +3,33 @@
 using BinaryBuilder, Pkg
 
 name = "oxigraph_server"
-version = v"0.2.5"
+version = v"0.3.18"
+
+url_prefix = "https://github.com/oxigraph/oxigraph/releases/download/v$version/oxigraph_server_v$version"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/oxigraph/oxigraph.git", "a21dcbb4f7355d7a00a86fbc5ad2c350a53629c4"),
+    FileSource("$(url_prefix)_aarch64_apple", "c0c2a64e7dc05cf9c24d4c29349baef583eead3e1f9984cdc2ac56a5beba9df7"; filename = "oxigraph_server-arm64-apple-darwin"),
+    FileSource("$(url_prefix)_x86_64_apple", "c5d1229d1011d30ed55226545abc9f9caa5f40d34cecf3b7d0e6964db516df6b"; filename = "oxigraph_server-x86_64-apple-darwin"),
+    FileSource("$(url_prefix)_aarch64_linux_gnu", "be0ec046fe48adff38e08c235d8f3f56af8d278a0a672c7f438ab32504ecaa57"; filename = "oxigraph_server-aarch64-linux-gnu"),
+    FileSource("$(url_prefix)_x86_64_linux_gnu", "1d62d475516f85dc8ab548ed0f8d25572186cbaf91cf43805d415d310045ea1e"; filename = "oxigraph_server-x86_64-linux-gnu"),
+    FileSource("$(url_prefix)_x86_64_windows_msvc.exe", "232fab182aa30df0d004980b6d386fab3aabb89ec294386ee63df0f56622ccf1"; filename = "oxigraph_server-x86_64-w64-mingw32"),
+    FileSource("https://raw.githubusercontent.com/oxigraph/oxigraph/v$version/LICENSE-MIT", "1f4f6736adc52ebfda18bb84947e0ef492bd86a408c0e83872efb75ed5e02838"; filename = "LICENSE.txt")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/oxigraph/server
-
-cargo build --release --no-default-features --features=sled
-
-install_license $WORKSPACE/srcdir/oxigraph/LICENSE-MIT
-
-cp ../target/${rust_target}/release/oxigraph_server${exeext} ${bindir}/
+cd ${WORKSPACE}/srcdir/
+mkdir -p "${bindir}"
+mv oxigraph_server-${target} oxigraph_server${exeext}
+cp oxigraph_server${exeext} ${bindir}
+chmod +x ${bindir}/*
```

Comment:
You can do all of this with a single command
```suggestion
install -Dvm 755 "oxigraph_server-${target}" "${bindir}/oxigraph_server${exeext}"
```

Reply:
Awesome!